inbm/dispatcher-agent/tests/unit/source/test_source_command.py

```diff
@@ -163,7 +163,7 @@ def test_do_source_command_remove(
             "dispatcher.source.source_command.create_application_source_manager",
             OsType.Ubuntu,
             ApplicationAddSourceParameters(
-                file_name="repofilename",
+                source_list_file_name="repofilename",
```

Comment:
**Maintainability**
**Suggestion:** The parameter name source_list_file_name should be consistent across all instances. If the change to source_list_file_name is intentional and the class ApplicationAddSourceParameters has been updated accordingly, ignore this suggestion. Otherwise, revert to the original parameter name file_name for consistency.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            ApplicationAddSourceParameters(
                file_name="repofilename",
                gpg_key_name="keyname",
                gpg_key_uri="gpguri",
                sources=["sourceA", "sourceB"],
            ),
```